packages/cli/src/utils/sentry.utils.ts

```diff
@@ -49,21 +44,31 @@ export const wrapHandler = function wrapHandler_<T>(
         const transaction = Sentry.getCurrentHub().getScope()?.getTransaction();
         if (transaction !== undefined) {
           transaction.setStatus("ok");
+          transaction.finish();
         }
       })
       .catch(async (error) => {
         await reportHandlerError(error);
         const transaction = Sentry.getCurrentHub().getScope()?.getTransaction();
         if (transaction !== undefined) {
           transaction.setStatus("unknown_error");
-        }
-        throw error;
-      })
-      .finally(() => {
-        const transaction = Sentry.getCurrentHub().getScope()?.getTransaction();
-        if (transaction !== undefined) {
           transaction.finish();
         }
+
+        // Don't display the help text which can obscure the error
+        process.exit(1);
       });
   };
 };
+
+const reportHandlerError: (error: unknown) => Promise<void> = async (error) => {
+  const logger = log.getLogger(METICULOUS_LOGGER_NAME);
+  logger.info("");
+  logger.error(error instanceof Error ? error.message : error);
+  logger.info("");
+  logger.info(
+    "Tip: run `meticulous help <command>` for help on a particular command, or `meticulous help` for a list of the available commands."
```

Comment:
Previously it'd post the full help message which could be quite long and made it harder to spot the error.